accounts-db/src/accounts_db.rs

```diff
@@ -4341,11 +4341,20 @@ impl AccountsDb {
                 // clean needs to take care of this dead slot
                 self.accounts_index.add_uncleaned_roots([slot]);
             }
-            info!(
-                "Unexpected shrink for slot {} alive {} capacity {}, \
-                likely caused by a bug for calculating alive bytes.",
-                slot, shrink_collect.alive_total_bytes, shrink_collect.capacity
-            );
+
+            if !shrink_collect.all_are_zero_lamports {
+                // if all are zero lamports, then we expect that we would like to mark the whole slot dead, but we cannot. That's clean's job.
+                info!(
+                    "Unexpected shrink for slot {} alive {} capacity {}, \
+                        likely caused by a bug for calculating alive bytes. \
+                        All alive bytes are zero: {}, {}",
```

Comment:
we can get rid of the last 2 args.

Reply:
backing up, since shrink treats single ref zero lamport accounts as dead, we can now correctly hit `num_alive_bytes`=0. This logs and says we have a bug. But, we don't. So, we need to silence that misleading info.